### DISCUSSION TITLE
Facing 404 Error on http://localhost:6060/ After Installing Clair

### DISCUSSION THREAD

**Discussion by Emoter1979** [quay/clair#1775] [Q&A]
Hello everyone,

I am currently facing an issue after following the steps outlined in "Getting Started with Clair" and "Testing Clair" sections of "the book". After successfully running the installation process, I encountered the following results:

```
Running 5/5
✔ Network clair_default Created                                                                                                                            0.1s
✔ Container clair-database Healthy                                                                                                                        7.6s
✔ Container clair-matcher Started                                                                                                                        8.0s
✔ Container clair-indexer Started                                                                                                                        8.0s
✔ Container clair-traefik Started
```

Although Traefik is running, when I try to access `http://localhost:6060/`, I am receiving a 404 error.

I have double-checked the installation steps and made sure that all the containers are up and running. However, I am unable to access the expected web page.

I would greatly appreciate any assistance or guidance on how to resolve this issue. Thank you in advance for your help!

Best regards,
Me

**Comment by hdonnay** [comment]
Clair doesn't serve any api endpoints at `/`. Take a look at the OpenAPI spec for the endpoints.
